src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -930,9 +980,10 @@ static PyObject *TopicPartition_str0 (TopicPartition *self) {
 
 	ret = cfl_PyUnistr(
 		_FromFormat("TopicPartition{topic=%s,partition=%"CFL_PRId32
-			    ",offset=%s,error=%s}",
+			    ",offset=%s,leader_epoch=%s,error=%s}",
```

Comment:
Just a thought - Do we really need to print leader_epoch everytime? I think if it is none, we shouldn't print.